k8s/apps/django-mindjunkies-web.yml

```
@@ -0,0 +1,46 @@
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: django-mindjunkies-web-deployment
+  template:
+    metadata:
+      labels:
+        app: django-mindjunkies-web-deployment
+    spec:
+      containers:
+      - name: django-mindjunkies-web
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)